test/utils_test.go

```diff
@@ -502,7 +502,7 @@ func recoverRouter(conf *node_config.RouterNodeConfig, logger *zap.SugaredLogger
 	configUpdateProposer := &policyMocks.FakeConfigUpdateProposer{}
 	configUpdateProposer.ProposeConfigUpdateReturns(nil, nil)
 
-	router := router.NewRouter(conf, logger, fakeSigner, configUpdateProposer)
+	router := router.NewRouter(conf, 0, logger, fakeSigner, configUpdateProposer)
```

Comment:
it will work for now but here the recover is used after a reconfig in the test so the config seq should change as well